yarn-project/foundation/src/eth-public-key/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './fr.js';
```

Comment:
Could you re export this file from aztec-rpc/src/index.ts so we can import it in the e2e test through aztec.js instead of directly from foundation?